src/box/lua/misc.cc

```diff
@@ -61,6 +63,29 @@ lbox_encode_tuple_on_gc(lua_State *L, int idx, size_t *p_len)
 	return (char *) region_join_xc(gc, *p_len);
 }
 
+API_EXPORT const char *
+position_pack_on_gc(struct position *pos, uint32_t *size)
```

Comment:
Since `box_select_after` returns a packed position, this function won't be needed anymore.

Reply:
done